docs/readmes/basics/prerequisites.md

```diff
@@ -15,17 +15,19 @@ Install the following tools:
 1. [Docker](https://www.docker.com) and Docker Compose
 2. [Homebrew](https://brew.sh/) *only* for MacOS users
 3. [VirtualBox](https://www.virtualbox.org/)
-3. [Vagrant](https://vagrantup.com)
+4. [Vagrant](https://vagrantup.com)
 
 Replace `brew` with your OS-appropriate package manager as necessary, or see
 the [pyenv installation instructions](https://github.com/pyenv/pyenv#installation).
 
 ```bash
+brew install go@1.13
 brew install pyenv
 
 # Replace .zshrc with your appropriate shell RC file
 # IMPORTANT: Use .bash_profile, not .bashrc for bash
-echo -e 'if command -v pyenv 1>/dev/null 2>&1; then\n  eval "$(pyenv init -)"\nfi' >> ~/.zshrc
+echo export PATH="/usr/local/opt/go@1.13/bin:$PATH" >> ~/.zshrc
```

Comment:
This is also not *quite* right. When you don't single-quote the arguments to `echo`, the `$PATH` variable gets expanded *when you run the echo command* rather than when the shell file is sourced [(ref)](https://www.howtogeek.com/howto/29980/whats-the-difference-between-single-and-double-quotes-in-the-bash-shell/)

Consider the following, where the latter is the desired behavior

```
~/Desktop/tmp                                                                                                                             
$ echo -n >| tmp.txt                                                            

~/Desktop/tmp                                                                                                                             
$ echo export PATH="/usr/local/opt/go@1.13/bin:$PATH" >> tmp.txt ; cat tmp.txt  
export PATH=/usr/local/opt/go@1.13/bin:/Users/hcgatewood/.pyenv/shims:/usr/local/opt/go@1.13/bin:/usr/local/sbin:/usr/local/bin:/opt/facebook/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/TeX/texbin:/opt/facebook/bin/biggrep:/opt/facebook/ops/scripts/chef:/opt/homebrew/bin:/usr/local/munki:/opt/facebook/hg/bin:/Library/Apple/usr/bin:/Applications/Wireshark.app/Contents/MacOS:/Users/hcgatewood/.scripts:/Users/hcgatewood/Desktop:/Users/hcgatewood/go/bin:/usr/local/go/bin:/Users/hcgatewood/.npm-global/bin:/Users/hcgatewood/.local/bin:/usr/local/opt/fzf/bin:/Users/hcgatewood/.rvm/bin

~/Desktop/tmp                                                                                                                             
$ echo -n >| tmp.txt                                                          

~/Desktop/tmp                                                                                                                             
$ echo 'export PATH="/usr/local/opt/go@1.13/bin:$PATH"' >> tmp.txt ; cat tmp.txt
export PATH="/usr/local/opt/go@1.13/bin:$PATH"
```

Reply:
Got it! Thanks for the explanation Hunter, I actually found that out the hard way when I couldn't get it working. I managed to figure it was due to the missing single quotes later on but was manually adding it when I typed it in shell and forgot to add it to the commit.